js/lib/beautify.js

```diff
@@ -170,6 +170,27 @@
         var beautifier = new Beautifier(js_source_text, options);
         return beautifier.beautify();
     }
+    
+    function verifyOperatorPosition(opPosition) {
+        var validPositionValues = [undefined];
+        Object.keys(OPERATOR_POSITION).forEach(function(pos) {
```

Comment:
Ha - I had no idea IE had trouble with that.  I'll implement it without the polyfill.